src/rpc/net.cpp

```diff
@@ -974,6 +974,60 @@ static RPCHelpMan addpeeraddress()
     };
 }
 
+static RPCHelpMan addrmaninfo()
```

Comment:
realized it probably makes more sense to call it `getaddrmaninfo` to be consistent with other RPC messages

Reply:
true, i've updated the PR to call it `getaddrmaninfo`.